misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -24,19 +24,19 @@ if command -v jq 2>&1 >/dev/null; then
 	done
 else
 	echo 'Error: Could not find 'jq' command. Is jq installed? Try running "brew install jq" or "port install jq" and rerunning this script.'
-	exit 0
+	exit 1
 fi
 
 # Download and install the Vulkan SDK.
 curl -L "https://sdk.lunarg.com/sdk/download/latest/mac/vulkan-sdk.zip" -o /tmp/vulkan-sdk.zip
 unzip /tmp/vulkan-sdk.zip -d /tmp
 
-if [ -d "/tmp/InstallVulkan-$new_ver_full.app" ]; then
-	/tmp/InstallVulkan-$new_ver_full.app/Contents/MacOS/InstallVulkan-$new_ver_full  --accept-licenses --default-answer --confirm-command install
-	rm -rf /tmp/InstallVulkan-$new_ver_full.app
-elif [ -d "/tmp/InstallVulkan.app" ]; then
-	/tmp/InstallVulkan.app/Contents/MacOS/InstallVulkan --accept-licenses --default-answer --confirm-command install
-	rm -rf /tmp/InstallVulkan.app
```

Comment:
I dropped previous formats as we always download `latest` anyway and I suppose they won't change back to a previous format again.

We could re-add those if we want to add support for specifying the desired SDK version to download.

BTW, since the version is part of the `.app` name now, we could drop the `jq` requirement and just extract it from there.